public/data/departments.json

```diff
@@ -1 +1 @@
-{"0":"Agricultural Extension","1":"Agricultural Extension","2":"Arts Commission","3":"Assessor of Property","4":"Beer Board","5":"Circuit Court Clerk","6":"Circuit Court Satellite","7":"Clerk and Master","8":"Codes Administration","9":"Community Ed Commission","10":"Convention Center Authority","11":"County Clerk","12":"Criminal Court Clerk","13":"Criminal Justice Planning Unit","14":"District Attorney","15":"Election Commission","16":"Emergency Communication Center","17":"Farmer's Market","18":"Finance","19":"Fire","20":"General Services","21":"General Sessions Court","22":"Health","23":"Historical Commission","24":"Human Relations Commission","25":"Human Resources","26":"Information Technology Service","27":"Internal Audit","28":"Justice Integration Services","29":"Juvenile Court","30":"Juvenile Court Clerk","31":"Law","32":"Mayor's Office","33":"Metro Action Commission","34":"Metropolitan Clerk","35":"Metropolitan Council","36":"MTA","37":"Municipal Auditorium","38":"NCAC","39":"Office of Emergency Mgmt","40":"Parks","41":"Planning Commission","42":"Police","43":"Public Defender","44":"Public Library","45":"Public Works","46":"Register of Deeds","47":"Sheriff","48":"Social Services","49":"Soil and Water Conservation","50":"Sports Authority","51":"State Fair Board","52":"State Trial Courts","53":"Trustee","54":"Water Services","55":"Administrative","56":"Airport Authority","57":"Office of Family Safety","58":"Emergency Communications","59":"Community Oversight Board"}
```

Comment:
I wonder why we lost the Community Oversight Board, maybe just because you were using older data?

Reply:
This is an odd one - I didn't notice this before.  I looked at the data files, and I see the Community Oversight Board has 10, or more, employees listed in each of those files.  Not sure why they didn't make it to this file.  This file, like the others, is produced by the data-import script - just a quick look at the script, the logic seems to work, so I'm dumbfounded - might need to do a step by step debug on that data-import script to be sure it's working the way it should.

Reply:
Would you be willing to dig into this a bit more? This is the only blocking concern in my mind. I've made a separate issue to fix the data discrepancies here: https://github.com/code-for-nashville/open-data-portal/issues/10. I'm not worried about those because we can just fix them in our open data portal and they'll pull through to here.

Meanwhile, I want to update the website, because we're a year out of date, but I think we should fix this bug or else it will go on forever.

Reply:
Actually, I think this was a preexisting issue, your changes just exposed it because you brought in files from the city that didn't match the exact column name expected (see: https://github.com/code-for-nashville/inclucivics/issues/188)